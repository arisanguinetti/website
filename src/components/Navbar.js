import React, { Component } from "react";

class Navbar extends Component {
  render() {
    return (
      <nav
        id="navigation"
        className="navbar navbar-fixed-top center-menu"
        role="navigation"
      >
        <div className="container navbar-container">
          <div className="navbar-header page-scroll">
            <button
              type="button"
              className="navbar-toggle"
              data-toggle="collapse"
              data-target=".berg-collapse"
            >
              <span className="sr-only">Toggle navigation</span>
              <span className="icon-bar" />
              <span className="icon-bar" />
              <span className="icon-bar" />
            </button>

            <a className="navbar-brand visible-xs visible-sm" href="#page-top">
              Ari Sanguinetti
            </a>
          </div>

          <div className="navbar-collapse collapse berg-collapse">
            <ul className="nav navbar-nav">
              <li className="page-scroll">
                <a href="#home">Home</a>
              </li>

              <li className="page-scroll">
                <a href="#profile">Profile</a>
              </li>

              <li className="page-scroll">
                <a href="#strengths">Strengths</a>
              </li>

              <li className="page-scroll">
                <a href="#contact">Contact</a>
              </li>
            </ul>
          </div>
        </div>
      </nav>
    );
  }
}

export default Navbar;
