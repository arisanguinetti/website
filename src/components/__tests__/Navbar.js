import React from "react";
import { shallow } from "enzyme";
import Navbar from "../Navbar";

describe("Navbar", () => {
  describe("Rendering", () => {
    it("should match to snapshot", () => {
      const component = shallow(<Navbar />);
      expect(component).toMatchSnapshot();
    });
  });
});
