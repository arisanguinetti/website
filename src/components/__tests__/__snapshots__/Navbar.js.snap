// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Navbar Rendering should match to snapshot 1`] = `
<nav
  className="navbar navbar-fixed-top center-menu"
  id="navigation"
  role="navigation"
>
  <div
    className="container navbar-container"
  >
    <div
      className="navbar-header page-scroll"
    >
      <button
        className="navbar-toggle"
        data-target=".berg-collapse"
        data-toggle="collapse"
        type="button"
      >
        <span
          className="sr-only"
        >
          Toggle navigation
        </span>
        <span
          className="icon-bar"
        />
        <span
          className="icon-bar"
        />
        <span
          className="icon-bar"
        />
      </button>
      <a
        className="navbar-brand visible-xs visible-sm"
        href="#page-top"
      >
        Ari Sanguinetti
      </a>
    </div>
    <div
      className="navbar-collapse collapse berg-collapse"
    >
      <ul
        className="nav navbar-nav"
      >
        <li
          className="page-scroll"
        >
          <a
            href="#home"
          >
            Home
          </a>
        </li>
        <li
          className="page-scroll"
        >
          <a
            href="#profile"
          >
            Profile
          </a>
        </li>
        <li
          className="page-scroll"
        >
          <a
            href="#strengths"
          >
            Strengths
          </a>
        </li>
        <li
          className="page-scroll"
        >
          <a
            href="#contact"
          >
            Contact
          </a>
        </li>
      </ul>
    </div>
  </div>
</nav>
`;
